README.rst

```diff
@@ -174,6 +174,18 @@ See `the contributing page <https://github.com/01org/cc-oci-runtime/blob/master/
 Building
 --------
 
+Acquiring
+~~~~~~~~~
+
+As ``cc-oci-runtime`` contains some golang code, the source tree needs to live
+in your ``$GOPATH`` in order to build. The easiest way to get the codebase into
+the correct place is with the command::
```

Comment:
For the sake of stop blocking this change, I will compromise. The other instances you mentioned were done before I was involved in the review process. Moving forward, added content should follow the best practices regardless of how it was done in the past. I agree that a separate PR is needed to fix all the previous instances.